js/enviro_phat.py

```diff
@@ -0,0 +1,36 @@
+''' Copyright 2017 Vprime
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.*/
+
+ @author Rami Alshafi <ralshafi@vprime.com> '''
+
+import sys
+import envirophat
```

Comment:
@alshafi can you add a comment here explaining that you need to install this python package?

Reply:
sure thing!